### PR TITLE
Chef Powercreep - More food is GOOD

### PR DESCRIPTION
**Pull request by MajManatee**
# Document the changes in your pull request

Have you ever been HUNGRY before?
Are you a HUMAN and typically subsist off of garbage from vending machines because the chef has made NO food?
Does the idea of just killing someone and eating their organs DISGUST you for some reason?

Well no longer shall you worry.

This PR adds more food to the round start chef containers, so they have even LESS of a reason to not do their jobs.

What it does:
Animal cube box now gives 1 extra cow because milk AND meat.
Flour up from 3 to 4 per containers.
Sugar up from 1 to 2 per container.
Rice up from 1 to 2 per container.
Eggs from 2 to 3 per container.

# Changelog

:cl:  
tweak: Chefs have more food components available shift start.
/:cl:

### CODE REVIEW

File: code/game/objects/items/storage/boxes.dm [hunk 1]
```diff
@@ -594,11 +594,11 @@
 	STR.can_hold = typecacheof(list(/obj/item/reagent_containers/food/snacks/monkeycube))
 
 /obj/item/storage/box/mixedcubes/PopulateContents()
-	for(var/i in 1 to 3)
```
Comment (ynot01) [inline code/game/objects/items/storage/boxes.dm:597]: Sir? This isn't randomization, this is a for loop

The animal farm box has 3 of each type

**Comment by ynot01** [comment]
Does the chef really need 150 units of sugar? He can easily get more and I have not once ever found myself run out

**Comment by Mantlecrawler** [comment]
> Does the chef really need 150 units of sugar? He can easily get more and I have not once ever found myself run out

50 Units is fairly limiting to me, especially with pastries

File: code/game/objects/items/storage/boxes.dm (hunk 1 above)
Comment (MajManatee) [reply code/game/objects/items/storage/boxes.dm:597]: i think im retarded

**Comment by patpol4** [comment]
This is stupid and makes cargo even more irrelevant, we need item scarcity, not this.

**Comment by MajManatee** [comment]
we HAVE scarcity, and becasue of that 4/5 shifts theres no god damned food.

**Comment by Marmio64** [comment]
one extra bag of rice and sugar, sure, perhaps even eggs. But flour, milk, and meat is very easy for botany to get you, since even if they're of intermediate experience they should be getting you near infinite biomass and thus near infinite of those 3 things.

**Review by ToasterBiome — CHANGES_REQUESTED** [review]
Too much, maybe between the current amount and these new amounts. The point is to give them enough leg room to start but not make it trivial for the entire round.

**Comment by MajManatee** [comment]
> one extra bag of rice and sugar, sure, perhaps even eggs. But flour, milk, and meat is very easy for botany to get you, since even if they're of intermediate experience they should be getting you near infinite biomass and thus near infinite of those 3 things.

The issue i have been informed of is when there ISNT a botanist, or when they dont want to co-operate, and you are just more or less fucked, hence the large volume.

**Comment by MajManatee** [comment]
> Too much, maybe between the current amount and these new amounts. The point is to give them enough leg room to start but not make it trivial for the entire round.

Reduced down by one across the board.

**Review by ToasterBiome — APPROVED** [review]

File: code/game/objects/items/storage/boxes.dm [hunk 2]
```diff
@@ -597,9 +597,10 @@
 	for(var/i in 1 to 3)
 		new /obj/item/reagent_containers/food/snacks/monkeycube/goat(src)
 		new /obj/item/reagent_containers/food/snacks/monkeycube/sheep(src)
-		new /obj/item/reagent_containers/food/snacks/monkeycube/cow(src)
 		new /obj/item/reagent_containers/food/snacks/monkeycube/chicken(src)
-
+	for(var/i in 1 to 4)
```
Comment (ReddicusDragon) [inline code/game/objects/items/storage/boxes.dm:601]: is this a valid way to do a for loop

Comment (Moltijoe) [reply code/game/objects/items/storage/boxes.dm:601]: yes

**Comment by Moltijoe** [comment]
council approved, good enough for me

**Pull request merged**